EULA.md

```diff
@@ -0,0 +1,288 @@
+WARRANTIES NOT EXPLICITLY GRANTED HEREIN, INCLUDING, BUT NOT LIMITED TO,
+ANY IMPLIED WARRANTIES OF FITNESS, MERCHANTABILITY, OR NON-INFRINGEMENT.
+
+8. **Personal Data and Other Confidential Information**
+
+To the extent your use of the Services requires it, you are responsible
+for providing notice to, and obtaining consents from, individuals
+regarding the collection, processing, transfer and storage of their data
+through your use of the Services.
+
+You understand and agree that you are solely responsible for any
+liabilities, fines and penalties arising from your failure to comply
+with this Section 8 and any other provision of this Agreement.
+
+9. **Telemetry Data**
+
+For the purposes of this Agreement, “telemetry data” shall mean any
+information related to your utilization of the Services.
+
+You acknowledge and agree that the Services will collect and transmit
+telemetry data to MOC Alliance and its higher education, non-profit and
+industry MOC Alliance Collaborators. MOC Alliance and its MOC Alliance
+Collaborators may use telemetry data to operate and maintain (including
+to improve) the Services and for scientific research purposes.
+
+Telemetry data will generally not include personal, identifiable
+information. You acknowledge that some of the telemetry data, such as IP
+address, MAC address and email address, may qualify as personal data
+identifying an individual under the laws of some jurisdictions.
+
+Requests by third parties other than Provider and MOC Alliance
+Collaborators to access telemetry data for any purpose must be reviewed
+and approved by the MOC Alliance Data Governance Committee. MOC Alliance
+may also post and make broadly available some telemetry data in an
+anonymized manner on the Internet.
+
+10. **Limitation of Liability**
+
+The Provider will use reasonable security measures to protect content
+stored on the Services. However, the Provider makes no guarantees that
+the Services will be free from loss, corruption, attack, viruses,
+interference, hacking, or other security intrusion, and the Provider
+disclaims any liability relating thereto.
+
+AS PERMITTED BY LAW, THE PROVIDER WILL NOT BE RESPONSIBLE FOR ANY LOST
+PROFITS, REVENUES, OR DATA, FINANCIAL LOSSES OR INDIRECT, SPECIAL,
+CONSEQUENTIAL, EXEMPLARY, OR PUNITIVE DAMAGES CAUSED BY OR IN ANY WAY
+ATTRIBUTABLE TO USE OF THE SERVICES. FOR THE SAKE OF CLARITY, THE
+PROVIDER WILL NOT BE RESPONSIBLE FOR THE LOSS OR BREACH OF YOUR USER
+CONTENT.
+
+11. **Provider’s Right to Update Services**
+
+The Provider reserves the right at any time to modify this Agreement and
+to impose new or additional terms or conditions on your use of the
+Services. Any material modifications and additional terms and conditions
+will be communicated to you and, if accepted, will be effective
+immediately and will be incorporated into this Agreement. In the event
+that you refuse to accept such changes, the Provider will have the right
```

Comment:
This wording might need to be stronger. Do we need to be clear that if they don't accept the changes they must terminate the agreement? The current wording sounds like we have to be the ones to first find out if they accepted and second terminate if they did not.
Something along the lines of "... will be communicated to you and, will be effective immediately and incorporated into this Agreement. In the event that you refuse to accept such changes, you are required to terminate this Agreement pursuant to Section 12.

Reply:
Follow up discussion with Kim: we want to, if it is ok to, put onus on them to cancel since they  may not be forced through log in which would require them to "sign off" for long periods of time as they may interface via programs only.

Reply:
addressed by a change to lines 15-19, now 15-22